### ISSUE TITLE
Onboard OmniXtend as graduated project

### ISSUE THREAD

**Issue by mgielda** [chipsalliance/tsc#47]
Michael asked Srini for guidance as to who we should contact.

**Comment by mgielda** [comment]
Adding @ZvonimirBandic as assignee so that he can help push this forward :)

The process is described here: https://github.com/chipsalliance/tsc/tree/main/projects

**Comment by mgielda** [comment]
Hi @dejan-vucinic could you take over here? YOu essentially have to create an application like this: https://github.com/chipsalliance/tsc/blob/main/projects/openfasoc.yml

And then we will vote on it.

**Comment by dejan-vucinic** [comment]
Oh, I just realized this thread exists.
https://www.goodreads.com/quotes/379100-there-s-no-point-in-acting-surprised-about-it-all-the

I'm a bit confused how to create the application, just add a file like that to git?

Also, can someone explain how do we "transfer" the OmniXtendEndpoint repository? Is it a rename or a copy?

**Comment by tgorochowik** [comment]
> I'm a bit confused how to create the application, just add a file like that to git?

Yes, the easiest way to do this is to create a fork of this repository and [contribute a PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) with a new yml file based the [template](https://github.com/chipsalliance/tsc/blob/main/projects/templates/graduated.yml).

After that, this topic will be discussed on the next TSC meeting and the feedback will be visible in the PR. You can also join the next meeting if you wish to join the discussion.

> Also, can someone explain how do we "transfer" the OmniXtendEndpoint repository? Is it a rename or a copy?

Github has an option to transfer a repository to another organization, let's worry about this after the project is voted in.

**Comment by dejan-vucinic** [comment]
OK, I created a pull request for omnixtend.yml, let me know if something incorrect

**Comment by mgielda** [comment]
Update: the vote passed, the project is in. However, we still need the repo transfer, so changing the title here and waiting for @dejan-vucinic to close on this before closing the issue.
